tests/test_other.py

```diff
@@ -9127,3 +9127,30 @@ def test_safe_heap_log(self):
     if not self.is_wasm_backend():
       self.set_setting('WASM', 0)
       self.do_run(src, 'SAFE_HEAP load: ')
+
+  @only_wasm_backend('iprintf/__small_printf are wasm-backend-only features')
+  def test_mini_printfs(self):
+    def test(code):
+      with open('src.c', 'w') as f:
+        f.write('''
+          #include <stdio.h>
+          int main() {
+            volatile void* unknown_value;
+            %s
+          }
+        ''' % code)
+      run_process([PYTHON, EMCC, 'src.c', '-O1'])
+      return os.path.getsize('a.out.wasm')
```

Comment:
do we need to delete src.c at the end?

Reply:
No, when the test ends the temp dir it ran in will be cleaned up.